src/main/scala/com/fulcrumgenomics/util/GeneAnnotations.scala

```diff
@@ -114,4 +115,84 @@ object GeneAnnotations {
     require(start <= end, "start is greater than end when creating an exon")
     def length: Int = end - start + 1
   }
+
+  /** Trait that gene biotypes will extends */
+  sealed trait GeneBiotype extends EnumEntry {
+    /** The biotype name in the GFF */
+    def key: String
+    /** True if this biotype represents a protein coding feature */
+    def isCoding: Boolean
+  }
+
+  /** Enum to represent gene biotypes in a GFF */
+  object GeneBiotype extends FgBioEnum[GeneBiotype] {
+    /** All values of the GeneBiotype Enum */
+    def values: IndexedSeq[GeneBiotype] = findValues
+
+    /** Allows GeneBiotypes to be built from the Enum name of the biotype name from the GFF */
+    override def apply(str: String): GeneBiotype = values.find(_.key == str).getOrElse(super.apply(str))
+
+    // antisense_RNA -> exon
```

Comment:
What do these comments mean?

Reply:
I think these were Seth's notes from the structure of entries below the gene in the RefSeq GFF.  They don't make sense here so I'll pull them out.